metron-interface/metron-alerts/src/app/alerts/alert-details/alert-details.component.ts

```diff
@@ -260,9 +260,7 @@ export class AlertDetailsComponent implements OnInit {
         commentRequest.timestamp = deletedCommentWrapper.alertComment.timestamp;
         commentRequest.sensorType = this.alertSourceType;
         this.updateService.removeComment(commentRequest).subscribe(
-            () => {
-              this.alertCommentsWrapper.map(alertsWrapper => alertsWrapper.alertComment)
-            },
+            null,
```

Comment:
I removed the next callback here because it makes no sense to map over an array and does nothing with the mapped array. 

[map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#Return_value) does not mutate the array on which you call it. It creates a brand new array with the mapped elements. The code above does nothing with the return value of map so it can be removed.

Reply:
Yeah, that makes sense to remove since we do not know the original intention of the line of code.

@justinleet It looks like this line of code was added here. Is it okay we remove it? Asking because I'm not sure what the original intention of the code was. Since there isn't a PR attached I'm not even sure you were the author of the code: 
https://github.com/apache/metron/commit/e0fc47582e65e9daf1a0c85dc1b1657166ff14f5#diff-bce5ae5c6be579250a35b6ff4f6da594R271